Shipr-Helm/templates/redis-pvc.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{  .Values.redis.name }}-pvc
+spec:
+  storageClassName: {{ . .Values.redis.storageClassName }}
+  accessModes:
+    {{- range .Values.redis.volume.accessModes }}
+    - {{ . }}
+    {{- end }}
+  resources:
+    requests:
+      storage: {{  .Values.redis.volume.storage }}
```

Comment:
No storageClassName, no selector, no annotations and no labels?

Reply:
Fixed